Misc/NEWS.d/next/Library/2022-10-24-11-01-05.gh-issue-98253.HVd5v4.rst

```diff
@@ -0,0 +1,2 @@
+Fixed a source of potential type-related refleaks caused by LRU caches in
```

Comment:
This doesn't match the investigation results. Neither typing, nor the lru_cache is broken. Neither has "caused refleaks".  Instead, the results of the investigation was that uncleared caches in typing would amplify existing refleaks in third-party extension modules.

This is more properly described as "harm reduction" rather than a bugfix.